pyQuARC/code/datetime_validator.py

```diff
@@ -116,7 +116,7 @@ def validate_ending_datetime_against_granules(ending_datetime, collection_shortn
         return DatetimeValidator.validate_datetime_against_granules(
             ending_datetime,
             collection_shortname,
-            'end_date',
+            '-end_date',
```

Comment:
why?

Reply:
guess it makes sense to do it here rather than at L92